content-services/content-sources-backend
# Fixes 2180: add index for tasks query

## Summary
A query of tasks such as `SELECT * FROM tasks WHERE org_id = '5894300' and repository_uuid = '7300282a-dc65-432b-bf20-52dc958b72f6' and status = 'running' and type = 'snapshot' ORDER BY tasks.id LIMIT 1"` is taking a while in stage.

## Testing steps
1. Run the db migration
2. Seed something like 100,000 tasks with a few different orgIDs and statuses.
3. Run a similar query to the one above. Add `explain` to the beginning i.e. `EXPLAIN SELECT...`
4. You should see in the output that the index is being used to make the query. 
5. You may not notice improved performance locally because the seeded data is already pretty well sorted.
